index.js

```diff
@@ -36,10 +36,17 @@ function isMatch(pattern, options) {
     };
   }
 
-  if (Array.isArray(pattern) || isObject(pattern)) {
+  if (Array.isArray(pattern)) {
+    return function (val) {
+      return mm(val, pattern, options).length !== 0;
```

Comment:
K, still seems like `.any()` makes more sense, and `.isMatch()` makes more sense on the string check. We're just returning a boolean. How is `!== 0` better, given that the function will always need to complete the loop before returning it's length?

See https://github.com/jonschlinkert/micromatch#any and https://github.com/jonschlinkert/micromatch#ismatch.



```js
mm.any('foo.md', ['